std/encoding/ignore.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+
+/**
+ * parse ingore parttern content and return partterns
```

Comment:
Typo "ingroe". 

This seems like a very specialized function that could be left inside the prettier directory. The jsdoc should include the string ".gitignore" otherwise it's confusing what it means.

Reply:
moved to `prettier/ignore.ts`

Reply:
Also `patterns` instead of `partterns`